### ISSUE TITLE
Maximize SlackModel

### ISSUE THREAD

**Issue by geoffroyleconte** [JuliaSmoothOptimizers/NLPModelsModifiers.jl#19]
When I use a maximization model, the `slack_meta` function automatically transform the problem to  minimization model.

**Comment by tmigot** [comment]
Do you have an example? It seems to me that it should work. I tried the following:
```
using ADNLPModels, NLPModels, NLPModelsModifiers, Test

x0 = zeros(10)
f = x -> sum(x)
meta = NLPModelMeta{Float64, Vector{Float64}}(10, x0 = x0, minimize = false)
nlp = ADNLPModel(meta, Counters(), ADNLPModels.ForwardDiffAD(length(x0)), f, x -> T[])
snlp = SlackModel(nlp)
@test snlp.meta.minimize == nlp.meta.minimize

F = x -> ones(2)
nls_meta = NLSMeta{Float64, Vector{Float64}}(
    2,
    10,
  )
nls = ADNLSModel(meta, nls_meta, NLSCounters(), ADNLPModels.ForwardDiffAD(length(x0)), F, x -> T[])
snls = SlackNLSModel(nls)
@test snls.meta.minimize == nls.meta.minimize

```

**Comment by geoffroyleconte** [comment]
This was fixed with 0.4, thank you!
